source/server/options_impl.cc

```diff
@@ -214,7 +214,7 @@ OptionsImpl::OptionsImpl(std::vector<std::string> args,
     mode_ = Server::Mode::InitOnly;
   } else {
     const std::string message = fmt::format("error: unknown mode '{}'", mode.getValue());
-    throwMalformedArgExceptionOrPanic(message);
+    throw MalformedArgvException(message);
```

Comment:
do these changes make sense or just a cleanup?

Reply:
the OrPanic macros are needed for E-M exceptionless build.  now this code is not used by E-M we can go back to the original throw.